collect_data.sh

```diff
@@ -36,6 +36,25 @@ function get_running_time_from_user {
 
 }
 
+function get_ip_xor_mask_from_user {
+        default_user_ip_xor_mask="0.0.0.0"
+        user_ip_xor_mask=""
+        validated_ip_mask=false
+        read -p "Please enter an ip mask for using to xor the original ips, for default (without xor the original ips) press enter: " user_ip_xor_mask
```

Comment:
```suggestion
        read -p "If you don't want to share your IPs  please enter a fake IP that will be used for xoring the original IPs in the output file, otherwise press enter: " user_ip_xor_mask
```